projects/packages/subscribers-dashboard/webpack.config.js

```diff
@@ -15,17 +15,7 @@ module.exports = [
 		optimization: {
 			...jetpackWebpackConfig.optimization,
 		},
-		resolve: {
-			...jetpackWebpackConfig.resolve,
-			alias: {
-				...jetpackWebpackConfig.resolve.alias,
-				'@automattic/calypso-config': '@automattic/calypso-config/src/client.js',
-			},
-			fallback: {
-				...jetpackWebpackConfig.resolve.fallback,
-				events: require.resolve( 'events/' ),
-			},
-		},
```

Comment:
I don't see a direct dependency on this, but I'm also a bit confused as to what this did and why it can now be removed.

Further irrelevant confusion...it seems the latest published version of `@automattic/calypso-config` is `1.0.0-alpha.0`:
https://www.npmjs.com/package/@automattic/calypso-config

But there appear to be a few newer tags:
https://github.com/Automattic/calypso-config